cuda/private/templates/BUILD.cufile

```diff
@@ -0,0 +1,9 @@
+exports_files(glob(["**"]))
+
+cc_library(
+    name = "cufile",
+    srcs = glob(["cufile/lib/**/*.so*"]),
```

Comment:
For some reason I don't quite remember, please use `cc_import_versioned_sos`, the https://github.com/bazel-contrib/rules_cuda/blob/4233145fc5e2ff78d871fc3e7264c58f6be5942e/cuda/private/templates/BUILD.curand can be a very simple template.